src/components/View/NewsItem/NewsItemSchemaOrg.jsx

```diff
@@ -0,0 +1,77 @@
+import { useIntl } from 'react-intl';
+import { toPublicURL } from '@plone/volto/helpers';
+import { SiteProperty } from 'volto-site-settings';
+import {
+  getSiteProperty,
+  SchemaOrgUtils,
+  richTextHasContent,
+  SchemaOrg,
+} from 'io-sanita-theme/helpers';
+
+const NewsItemSchemaOrg = ({ content }) => {
+  const intl = useIntl();
+
+  let siteTitle = SiteProperty({
+    property: 'site_title',
+    getValue: true,
+    defaultTitle: getSiteProperty('siteTitle', intl.locale),
+  });
+
+  siteTitle = siteTitle?.replaceAll('\\n', ' - ') ?? '';
+
+  let schemaOrg = {
+    '@type': 'NewsArticle',
+    headline: content.title,
+    url: toPublicURL(content['@id']),
+  };
+
+  let description = [];
+
+  if (content.description) {
+    description.push(content.description);
+  }
+
+  if (richTextHasContent(content.descrizione_estesa)) {
+    description.push(
+      SchemaOrgUtils.fieldDataToPlainText(content.descrizione_estesa),
+    );
+  }
+
+  if (description.length > 0) {
+    schemaOrg.description = description.join('. ');
+  }
+
+  if (Array.isArray(content.uo_correlata) && content.uo_correlata.length > 0) {
+    schemaOrg.author = content.uo_correlata.map((e) => {
+      const author = {
+        '@type': 'Organization',
+        name: e.title,
+      };
+
+      return author;
+    });
+  }
+
+  if (intl.locale) {
+    schemaOrg.inLanguage = intl.locale;
+  }
```

Comment:
```suggestion

    schemaOrg.inLanguage = content.language.token;

```

il valore è nel contenuto, usiamo quello